src/main/scripts/install.sh

```diff
@@ -32,122 +66,114 @@ apk add docker-cli
 
 # Install `kubectl` and connect to the AKS cluster
 az aks install-cli
-az aks get-credentials -g $resourceGroupName -n $aksClusterName --overwrite-existing
+az aks get-credentials -g $clusterRGName -n $clusterName --overwrite-existing >> $logFile
 
-# Install Open Liberty Operator V0.7
+# Install Open Liberty Operator V0.7.1
 OPERATOR_NAMESPACE=default
 WATCH_NAMESPACE='""'
-kubectl apply -f https://raw.githubusercontent.com/OpenLiberty/open-liberty-operator/master/deploy/releases/0.7.0/openliberty-app-crd.yaml
-curl -L https://raw.githubusercontent.com/OpenLiberty/open-liberty-operator/master/deploy/releases/0.7.0/openliberty-app-cluster-rbac.yaml \
-      | sed -e "s/OPEN_LIBERTY_OPERATOR_NAMESPACE/${OPERATOR_NAMESPACE}/" \
-      | kubectl apply -f -
-curl -L https://raw.githubusercontent.com/OpenLiberty/open-liberty-operator/master/deploy/releases/0.7.0/openliberty-app-operator.yaml \
-      | sed -e "s/OPEN_LIBERTY_WATCH_NAMESPACE/${WATCH_NAMESPACE}/" \
-      | kubectl apply -n ${OPERATOR_NAMESPACE} -f -
+kubectl apply -f https://raw.githubusercontent.com/OpenLiberty/open-liberty-operator/master/deploy/releases/0.7.1/openliberty-app-crd.yaml
+curl -L https://raw.githubusercontent.com/OpenLiberty/open-liberty-operator/master/deploy/releases/0.7.1/openliberty-app-cluster-rbac.yaml \
```

Comment:
Can we use a variable for the version number instead of hard coding it in the URLs?

Reply:
Good point, fixed in commit in a6499c2.